### PR TITLE
Add Pamphile Roy to Team page

### PR DESCRIPTION
**Pull request by tupui**
Adding myself to the Team page 😃

(I did not make a fork, not sure how you prefer to work here.)

### CODE REVIEW

**Comment by rgommers** [comment]
Thanks @tupui. Nikola is unhappy:
```
Nikola: ruamel.yaml.scanner.ScannerError: mapping values are not allowed in this context
```

> (I did not make a fork, not sure how you prefer to work here.)

Not a mortal sin, but generally prefer forks I'd say.

File: galleries/team/metadata.yml [hunk 1]
```diff
@@ -169,3 +169,8 @@ title: Cristian Garcia
 github: cgarciae
 projects: ['elegy', 'treex', 'lux']
 ---
+name: pamphile_roy.jpg
+ title: Pamphile Roy
```
Comment (rgommers) [inline galleries/team/metadata.yml:173]: 1 space indentation is the problem here I think

Comment (tupui) [reply galleries/team/metadata.yml]: Ooops thanks, it was the copy paste.

**Comment by tupui** [comment]
>  > (I did not make a fork, not sure how you prefer to work here.)
> 
> Not a mortal sin, but generally prefer forks I'd say.

ok I will next time 👍 At my previous companies they did not allow to make a fork 😅

**Review by rgommers — APPROVED** [review]
Thanks @tupui, LGTM now

**Pull request merged**